ddtrace/contrib/pymemcache/__init__.py

```diff
@@ -0,0 +1,17 @@
+"""Instrument pymemcache to report memcached queries.
+
+``patch_all`` will automatically patch the pymemcache client::
+
+    from ddtrace import Pin, patch
+    import pymemcache
+
+    # If not patched yet, patch pymemcache specifically
+    patch(pymemcache=True)
+
+    # This will report a span with the default settings
+    client = pymemcache.client.base.Client(('localhost', 11211))
+    client.set("my-key", "my-val")
+
+    # Use a pin to specify metadata related to this client
+    Pin.override(client, service='my-memcached-service')
```

Comment:
I would like to replace the `Pin` with our `Config` object as soon as possible. It solves a huge amount of issues with a simpler API.